packages/connectivity/src/scp-cf/destination/destination.ts

```diff
@@ -329,21 +338,71 @@ export interface DestinationJson {
  */
 export interface DestinationConfiguration {
   [key: string]: any;
+  /**
+   * `URL` of the destination.
+   */
   URL: string;
+  /**
+   * `Name` of the destination.
+   */
   Name?: string;
+  /**
+   * `ProxyType` of the destination e.g. `Internet` or `OnPremise`.
+   */
   ProxyType?: string;
+  /**
+   * `sap-client` defined in the destination.
+   */
   'sap-client'?: string;
+  /**
+   * Username in case of basic authentication destinations.
+   */
   User?: string;
+  /**
+   * Password in case of basic authentication destinations.
+   */
   Password?: string;
+  /**
+   * Represents the authentication type of a destination.
+   */
   Authentication?: AuthenticationType;
+  /**
+   * Value of the TrustAll property of the destination.
+   */
   TrustAll?: string;
+  /**
+   * URL of the token service endpoint to retrieve access token.
+   * This may contain placeholders in multi-tenant scenarios https://help.sap.com/docs/CP_CONNECTIVITY/cca91383641e40ffbe03bdc78f00f681/c69ea6aacd714ad2ae8ceb5fc3ceea56.html?locale=en-US
```

Comment:
```suggestion
   * This may contain placeholders in multi-tenant scenarios. @see {@link https://help.sap.com/docs/CP_CONNECTIVITY/cca91383641e40ffbe03bdc78f00f681/c69ea6aacd714ad2ae8ceb5fc3ceea56.html?locale=en-US|OAuth SAML Bearer Assertion Authentication}.
```